src/libs/modulegroups.c

```diff
@@ -23,6 +23,8 @@
 #include "control/conf.h"
 #include "control/control.h"
 #include "develop/develop.h"
+#include "dtgtk/button.h"
+#include "dtgtk/icon.h"
 #include "gui/gtk.h"
 #include "libs/lib.h"
 #include "libs/lib_api.h"
```

Comment:
Shouldn't this module have version upgraded to 2 ?

Reply:
We can, but as presets where no implemented in previous version, I'm not sure it's needed...